style/rails/README.md

```diff
@@ -28,9 +28,7 @@ Migrations
 
 * Set an empty string as the default constraint for non-required string and text
   fields. [Example][default example].
-* List timestamps first when creating a new table. [Example][timestamps example].
```

Comment:
I removed this line entirely (instead of replacing it with the opposite guideline) because, as the commit message mentions, `rails generate` puts timestamps at the bottom by default.